modules/ROOT/pages/design-documentation.adoc

```diff
@@ -0,0 +1,305 @@
+
+----
+openapi: '3.0.2'
+info:
+  title: Multi-server Cemo API
+  version: '1.0'
+servers: <1>
+  - url: https://{customerId}.saas-app.com:{port}/v2
+    variables:
+      customerId:
+        default: demo
+        description: Customer ID assigned by the service provider
+      port:
+        enum:
+          - '443'
+          - '8443'
+        default: '443'
+  - url: https://{region}.api.company.com
+    variables:
+      region:
+        default: westus
+        enum:
+          - westus
+          - eastus2
+          - westcentralus
+          - westeurope
+          - southeastasia
+  - url: https://api.company.org/data/2.5/
+    description: Production server
+  - url: http://beta.api.company.org/data/2.5/
+    description: Beta server
+paths:
+  /default:<2>
+    description: A default path to the API
+    get:<3>
+      summary: "The get method for /default"
+      responses:
+        200:
+          description: Successful response
+  /duplicated:<4>
+    description: Has some of the default servers
+    servers:
+      - url: https://api.company.org/data/2.5/
+        description: Production server
+      - url: http://beta.api.company.org/data/2.5/
+        description: Beta server
+    get:<5>
+      summary: "The get method for /duplicated"
+      responses:
+        200:
+          description: Successful response
+  /files:<6>
+    description: File upload and download operations
+    servers:
+      - url: https://files.example.com
+        description: Override base path for all operations with the /files path
+    get:<7>
+      summary: "The get method for /files"
+      responses:
+        200:
+          description: Successful response
+  /ping:<8>
+    servers:
+      - url: https://endpoint.example.com
+        description: Override base path servers
+    get:<9>
+      servers:
+        - url: https://echo.example.com
+          description: Override base path for the GET /ping operation
+        - url: https://echo2.example.com
+          description: Also overrides base path for the GET /ping operation
+      summary: "The get method for /ping"
+      responses:
+        200:
+          description: Successful response
+    post:<10>
+      summary: "The post method for /ping"
+      responses:
+        201:
+          description: Successful create
+----
+
+1. The Summary page of the documentation for this API lists the four servers that are defined in this `servers` field:
++
+image::apid-apic-oas-3-summary.png[]
+
+2. The `/default` endpoint does not override the base `servers` definition at the top of the API. On the Overview page for the endpoint, you can select from those four servers to see the URI for this endpoint at each:
++
+image::apid-apic-oas-3-default.png[]
+
+3. The `GET` method for the `/default` endpoint also inherits the four servers in the base `servers` definition. On the page for this method, you can select from those four servers to see the URI for this method at each:
```

Comment:
ditto prev.: "at each" what? "at each server"?

Reply:
Per the "font choices" entry in our style ref., HTTP error codes, error names, and operations are not formatted, other than being in ALLCAPS, per industry standard.
Use GET and related, GLOBALLY in this PR.
--

Reply:
(I don't know why this ^ happened.)